rest/api/v1/openapi.yml

```
@@ -2562,10 +2562,10 @@ components:
           pattern: "^((0x)?[0-9a-fA-F]+|(earliest|pending|latest))$"
           type: string
         data:
-          description: Hexadecimal method signature and encoded parameters. Up to 24656 bytes as at most 49152 hexidecimal digits plus optional leading 0x.
+          description: Hexadecimal method signature and encoded parameters. Up to 131072 bytes as at most 262144 hexadecimal digits plus optional leading 0x.
           example: "0x47f1aae7"
           format: binary
-          maxLength: 49154
+          maxLength: 262144
```

Need to account for 0x.

```suggestion
          maxLength: 262146
```